new-client/src/components/Window.js

```diff
@@ -281,9 +276,11 @@ class Window extends React.PureComponent {
   }
 
   close = (e) => {
-    const { onClose } = this.props;
+    const { onClose, globalObserver, title } = this.props;
     this.latestWidth = this.rnd.getSelfElement().clientWidth;
     if (onClose) onClose();
+
+    globalObserver.publish("window-close", title);
```

Comment:
Could be nice with a better naming here? Could core.closeWindow be a better idea? This would comply with the other event names.

Reply:
Like your suggestion. I will update.